[pickers] Let the field components handle their opening UI, and allow field editing on mobile pickers (#15671)
mui/mui-x: docs/data/date-pickers/custom-opening-button/custom-opening-button.md

:::

## Render the opening button at the start of the input

> You can use the `openPickerButtonPosition` on the `field` slot to set the opening button at the start of the input (on the left in lef-to-right direction, on the right in left-to-right direction):

Author: Could you describe what is inconsistent in this demo?
I'm not sure to follow you

> You can use the `openPickerButtonPosition` on the `field` slot to set the opening button at the start of the input (on the left in lef-to-right direction, on the right in left-to-right direction):

Reviewer: I meant that technically this demo has nothing explicitlt to do with `RTL` behavior, it's just the default to swap elements.
I would suggest omitting the overly specific details about orientation specifics. 🤔

> You can use the `openPickerButtonPosition` on the `field` slot to set the opening button at the start of the input (on the left in lef-to-right direction, on the right in left-to-right direction):

Reviewer 2: I am with Lukas on this one. Maybe rephrasing it like this would make sense?
```suggestion
You can use the `openPickerButtonPosition` on the `field` slot to position the opening button on the right or left side of the input:
```

> You can use the `openPickerButtonPosition` on the `field` slot to set the opening button at the start of the input (on the left in lef-to-right direction, on the right in left-to-right direction):

Author: OK for me :+1:
